anyway/widgets/suburban_widgets/sub_urban_widget.py

```diff
@@ -4,13 +4,13 @@
 
 
 class SubUrbanWidget(Widget):
-    def __init__(self, request_params: RequestParams, name: str):
+    def __init__(self, request_params: RequestParams):

```

Comment:
Consider adding "calc_widget_digest"

Reply:
The call was moved to `register` decorator